package.json

```diff
@@ -34,11 +34,13 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn lint && jest --onlyChanged",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
```

Comment:
People will have a hard time learning and remembering the conventional commit format. Can you add https://github.com/commitizen/cz-cli to help them through this process?